go/cmd/vtctldclient/command/movetables.go

```diff
@@ -0,0 +1,541 @@
+	}
+
+	// MoveTablesCancel makes a MoveTablesCancel gRPC call to a vtctld.
+	MoveTablesCancel = &cobra.Command{
+		Use:                   "cancel",
+		Short:                 "Cancel a MoveTables VReplication workflow.",
+		Example:               `vtctldclient --server localhost:15999 movetables --workflow commerce2customer --target-keyspace customer cancel`,
+		DisableFlagsInUseLine: true,
+		Aliases:               []string{"Cancel"},
+		Args:                  cobra.NoArgs,
+		RunE:                  commandMoveTablesCancel,
+	}
+
+	// MoveTablesComplete makes a MoveTablesComplete gRPC call to a vtctld.
+	MoveTablesComplete = &cobra.Command{
+		Use:                   "complete",
+		Short:                 "Complete a MoveTables VReplication workflow.",
+		Example:               `vtctldclient --server localhost:15999 movetables --workflow commerce2customer --target-keyspace customer complete`,
+		DisableFlagsInUseLine: true,
+		Aliases:               []string{"Complete"},
+		Args:                  cobra.NoArgs,
+		RunE:                  commandMoveTablesComplete,
+	}
+
+	// MoveTablesCreate makes a MoveTablesCreate gRPC call to a vtctld.
+	MoveTablesCreate = &cobra.Command{
+		Use:                   "create",
+		Short:                 "Create and optionally run a MoveTables VReplication workflow.",
+		Example:               `vtctldclient --server localhost:15999 movetables --workflow commerce2customer --target-keyspace customer create --source-keyspace commerce --cells zone1 --cells zone2 --tablet-types replica`,
+		SilenceUsage:          true,
+		DisableFlagsInUseLine: true,
+		Aliases:               []string{"Create"},
+		Args:                  cobra.NoArgs,
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			// Either specific tables or the all tables flags are required.
+			if !cmd.Flags().Lookup("tables").Changed && !cmd.Flags().Lookup("all-tables").Changed {
+				return fmt.Errorf("tables or all-tables are required to specify which tables to move")
+			}
+			if cmd.Flags().Lookup("cells").Changed { // Validate the provided value(s)
+				for i, cell := range moveTablesCreateOptions.Cells { // Which only means trimming whitespace
+					moveTablesCreateOptions.Cells[i] = strings.TrimSpace(cell)
+				}
+			}
+			if !cmd.Flags().Lookup("tablet-types").Changed {
+				moveTablesCreateOptions.TabletTypes = tabletTypesDefault
+			}
+			if _, ok := binlogdatapb.OnDDLAction_value[strings.ToUpper(moveTablesCreateOptions.OnDDL)]; !ok {
+				return fmt.Errorf("invalid on-ddl value: %s", moveTablesCreateOptions.OnDDL)
+			}
+			return nil
+		},
+		RunE: commandMoveTablesCreate,
+	}
+
+	// MoveTablesShow makes a GetWorkflows gRPC call to a vtctld.
+	MoveTablesShow = &cobra.Command{
+		Use:                   "show",
+		Short:                 "Show the details for a MoveTables VReplication workflow.",
+		Example:               `vtctldclient --server localhost:15999 movetables --workflow commerce2customer --target-keyspace customer show`,
```

Comment:
(i see they have aliases to their lowercase versions, but now my second question is "is it weird that the parent command is uppercase with alias to lowercase while the child is lowercase with alias to upper?")

Reply:
I was using lowercase for the command but then when building the docs realized that ALL of the existing commands started with an uppercase letter. Otherwise I would have stuck with everything being lowercase. I thought that the current mix made it uniform while also supporting / showing all lowercase.